### ISSUE TITLE
Allow landscape mode

### ISSUE THREAD

**Issue by Bubu** [lexica/lexica#205]
I just tried playing at a tablet sitting on a desk, that didn't work :).

I saw that this was recently removed. @pserwylo would you accept a contribution adding this back? Any pointers on what to look out for?

I've built a version which allows landscape mode, my observations so far:
* The start screen looks okay-ish as is, one minor improvement here would be that the LEXICA letters are spaced too far apart now and they should probably have a max-width
* ScoreActivity is fully usable. The "Missed Words" tab could probably make more efficient use of screen real-estate... but not sure if it's worth introducing another layout for this.
* GameActivity is broken in landscape and would need either: some (probably) easy adjustments to make everything fit into the screen or have another landscape layout which has the game board next to the list of used words. Any opinions?

**Comment by pserwylo** [comment]
Absolutely, would love to add it back, any contributions very welcome. As you've probably noticed the `GameActivity` is a lot more hardcoded and so would involve a bit more work. It doesn't make use of Android layout XML files like the rest of the app. The reason for this is that I thought it would require a lot more custom rendering and measurement to get working. Though in hindsight, it probably wouldn't be particularly difficult to go back to using an XML based layout if some of the key parts were extracted into sensible custom `View`s.

Alternatively, as you mentioned some adjustments could probably be made, similar to your recommendation for the main screen, whereby we check the aspect ratio and set some max and min widths for important elements.
